ember/app/cases/new/controller.js

```diff
@@ -7,29 +7,19 @@ import getRootFormsQuery from "caluma-portal-demo/gql/queries/get-root-forms";
 import getWorkflowQuery from "caluma-portal-demo/gql/queries/get-workflow";
 import { queryManager } from "ember-apollo-client";
 import { task, lastValue } from "ember-concurrency";
-import QueryParams from "ember-parachute";
 
-const queryParams = new QueryParams({
-  selectedForm: {
-    defaultValue: "",
-    replace: true,
-  },
-});
-
-export default class CaseNewController extends Controller.extend(
-  queryParams.Mixin
-) {
+export default class CaseNewController extends Controller {
+  queryParams = ["selectedForm"];
   @queryManager apollo;
   @service router;
-  @tracked selectedForm;
+  @tracked selectedForm = "";
 
   setup() {
     this.fetchForms.perform();
   }
 
   reset() {
-    this.resetQueryParams();
-    this.selectedForm = null;
+    this.selectedForm = "";
```

Comment:
`setup` and `reset` are `ember-parachute` hooks which will never get fired. Either use [resetController](https://api.emberjs.com/ember/5.2/classes/Route/methods/resetController?anchor=resetController) to setup the reset similar to the "setup clone" you've already done or find another way.